internal/server/config/config_test.go

```diff
@@ -250,7 +250,22 @@ func TestVerifyConfig(t *testing.T) {
 func TestDefaultMaxConditionValuationCost(t *testing.T) {
 	// check to make sure DefaultMaxConditionEvaluationCost never drops below an explicit 100, because
 	// API compatibility can be impacted otherwise
+	t.Run("max_condition_evaluation_cost_too_low", func(t *testing.T) {
+		cfg := DefaultConfig()
+		cfg.MaxConditionEvaluationCost = 99
+
+		err := cfg.Verify()
+		require.Error(t, err)
+	})
+	t.Run("max_condition_evaluation_cost_valid", func(t *testing.T) {
+		cfg := DefaultConfig()
+		cfg.MaxConditionEvaluationCost = 120
+
+		err := cfg.Verify()
+		require.NoError(t, err)
+	})
 	require.GreaterOrEqual(t, DefaultMaxConditionEvaluationCost, 100)
+	require.GreaterOrEqual(t, MaxConditionEvaluationCost(), uint64(100))
```

Comment:
```suggestion
	require.GreaterOrEqual(t, MaxConditionEvaluationCost(), DefaultMaxConditionEvaluationCost)
```

Reply:
Merged the two so the test now makes more sense.